src/main/steps/upload-documents/content.test.ts

```diff
@@ -1,47 +1,49 @@
+  additionalDocumentsHeading: 'Additional documents',
+  paragraph2: 'You may want to submit other documents, files, or evidence that provides the Tribunal with more information about your case.',
+  paragraph3: 'This will depend on the nature of your appeal, application or claim. Specific examples can be found in your completed tribunal form.',
+  uploadedDocumentsSubTitle: 'These may include:',
   uploadDocumentsBullets: [
     'copies of any relevant correspondence or documentation',
+    'audio or video recordings',
     'written statements',
     'related reports',
   ],
-  caseRelevancySubTitle: 'For each document uploaded you must state why it is relevant to the case.',
+  caseRelevancySubTitle: 'For each piece of additional information or evidence you are submitting, please state why it is relevant to your case.',
   fileUploadRequirementsLabel: 'File upload requirements',
   uploadHelpBullets: [
-    'File formats: MS Word, MS Excel, PDF, JPG, PNG, TXT, RTF',
+    'File formats: MS Word, MS Excel, PDF, JPG, PNG, TXT, RTF, MP4, MP3',
     'File size per document: up to 20 megabytes (MB)',
+    'File size per multimedia file: up to 500 megabytes (MB)',
     'Files cannot be password protected',
   ],
   guideText:
-    'You cannot upload executable (.exe) or zip files because of virus risks. If you delete an uploaded file, you also delete the file description.',
+    'You can\'t upload executable (.exe), zip or other archive files due to virus risks.',
   titleForFile: 'Select documents to upload',
   uploadButton: 'Upload file',
   documentListLabel:
-    'List the description of each document uploaded on separate lines. For example, "1. Witness statement requested by the d2"',
+    'Please explain why you feel this document is relevant to your case.',
   delete: 'Delete',
   documentInLanguage: 'Document',
   fileuploaded: 'Files uploaded',
   errorSummaryMessage: 'There is a problem',
   uploadAFile: 'Upload a file',
+  uploadFileHint: 'Once you have selected the file click "Upload file" to upload it.',
   errors: {
     documentUpload: {
-      noDocumentUploaded: 'Please choose a file.',
-      fileSize: `The file you uploaded is too large.
-              Maximum file size allowed is 20MB`,
-      multimediaFileSize: 'The file you uploaded is too large. Maximum file size allowed is 500MB for multimedia files',
-      fileFormat: `The file you uploaded is in the wrong format.
-              Upload your file again in the correct format`,
+      noInput: 'You cannot continue without providing additional information or a document',
+      fileSize: `File size exceeds the maximum permitted value. Please upload a file that is less than 20MB (documents) or less than 500MB (multimedia files)`,
+      fileFormat: `This service only accepts files in the formats - Ms Word, MS Excel, PDF, JPG, PNG, TXT, RTF, MP4, MP3`,
```

Comment:
For consistency should it be `MS Word, MS Excel` as in the list above L22

Reply:
Should we also list out BMP in the list as it is valid?

Reply:
Error messaging is consistent with dss submit case. Happy to ask if this should be changed.